Plausible/Sampleable.lean

```diff
@@ -242,6 +260,18 @@ section Samplers
 
 open SampleableExt
 
+instance instToExprSum [ToExpr α] [ToExpr β] : ToExpr (α ⊕ β) :=
```

Comment:
Note that the use of `levelZero` in this instance is incorrect, which means that mathlib will have to make a copy of this instance and the one below (using Mathlib's `Lean.ToLevel`)